starter_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# The starter script for each VM instance of VMN
+# It installs the VNCServer, sets its password, starts a server, and
+# initiates noVNC's Websocket to TCP proxy/bridge Websockify.
+
+# This URL accesses the metadata value of key "session-id" of the current
+# VM, it will be used as a password to connect to the VNCServer.
+PASSWRD=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/session-id -H "Metadata-Flavor: Google")
+export USER=/root
```

Comment:
What is this for?

Reply:
Makes sure that the Environment variable USER is present for the user that you're running the vncserver as